python/py-astropy-healpix/Portfile

```diff
@@ -4,19 +4,20 @@ PortSystem 1.0
 PortGroup python 1.0
 PortGroup github 1.0
 
-github.setup        astropy astropy-healpix 0.5 v
-name                py-${name}
+name                py-astropy-healpix
```

Comment:
This change seems unnecessary. What was wrong with using the name variable as before ?

Reply:
I'm dropping the GitHub port group for all but the older Python 2.7-3.5 subport because the way that Astropy affiliated packages use setuptools_scm does not play nicely with MacPorts.